aspnet/identity/overview/getting-started/introduction-to-aspnet-identity.md

```diff
@@ -93,7 +93,7 @@ ASP.NET Identity is implemented using the following procedure. The purpose of th
 
 1. Create an ASP.NET MVC application with Individual Accounts. You can use ASP.NET Identity in ASP.NET MVC, Web Forms, Web API, SignalR etc. In this article we will start with an ASP.NET MVC application.  
   
-    ![](introduction-to-aspnet-identity/_static/image1.png)
+    ![Image of new A S P dot Net project window](introduction-to-aspnet-identity/_static/image1.png)
```

Comment:
It's ASP, not `A S P`

Reply:
I had thought screen readers interpreted ASP as an actual word and would try to pronounce it that way as opposed to the spacing variation of A S P in which each letter gets pronounced? Am I incorrect in that?

Reply:
change made